src/FactoryTrait.php

```diff
@@ -191,14 +191,13 @@ public function factory($seed, $defaults = [], string $prefix = null): object
      * add prefix.
      *
      * Rules observed, in order:
-     *  - If class starts with "." then prefixing is always done.
-     *  - If class contains "\" prefixing is never done.
+     *  - If class starts with "./" then prefixing is always done.
+     *  - If class contains "\" or it is an anonymous class prefixing is never done.
      *  - If class (with prefix) exists, do prefix.
      *  - don't prefix otherwise.
      *
      * Example: normalizeClassName('User', 'Model') == 'Model\User';
      * Example: normalizeClassName(Test\User::class, 'Model') == 'Test\User'; # or as per "use"
-     * Example: normalizeClassName('Test/User', 'Model') == 'Model\Test\User';
```

Comment:
If this syntax with ```/``` is no more supported, then this PR is BC.

Reply:
Yes, if anyone uses wrong names...

Reply:
fixed in desc, added label

Reply:
is it ok to add `bug` label on PRs which fixes some bug?